File: backend/src/main/java/cohappy/backend/model/dto/response/GetNotificationDTO.java
package cohappy.backend.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetNotificationDTO {
    private String eventId;
    private String eventType;
    private String title;
    private String subtitle;
    private String timestamp;
    private byte[] imageBytes;
    private String userCode;
}

File: frontend/app/src/main/java/cohappy/frontend/client/dto/response/GetNotificationDTO.kt
package cohappy.frontend.client.dto.response

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class GetNotificationDTO(
    val eventId: String,
    val eventType: String,
    val title: String,
    val subtitle: String,
    val timestamp: String,
    val imageBytes: ByteArray? = null,
    val userCode: String? = null
)